addons/l10n_it_ddt/data/l10n_it_ddt_template.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+	 <template id="my_view_name" inherit_id="l10n_it_edi.account_invoice_it_FatturaPA_export">
+            <xpath expr='//DatiDDT[@name="DDT"]' position="after">
```

Comment:
Can we put a t-if if not ddt_id is defined on the invoice?

Reply:
it done.